src/rubintv/models/models_assignment.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from typing import Any
+
+import yaml
```

Comment:
This import appears to be unused. But more than that, that scares me because linters etc should be catching things like that, so I'm more concerned that tests aren't failing because of that than the fact that this is here itself, if you see what I mean.

Reply:
Which one is unused?

Reply:
None, I am an idiot, please ignore 🙃